dcpy/test/connectors/edm/test_recipes.py

```diff
@@ -112,21 +156,63 @@ def test_parse_config():
     assert config.execution_details
 
 
-def test_get_config(load_library_parquet: library.Config):
-    config = recipes.get_config(TEST_LIBRARY_DATASET, load_library_parquet.version)
+def test_get_library_config(load_library_parquet: library.Config):
+    config = recipes.get_config(TEST_DATASET, load_library_parquet.version)
     assert config == load_library_parquet
 
 
-def test_get_latest_version(load_library_parquet: library.Config):
-    assert (
-        recipes.get_latest_version(TEST_LIBRARY_DATASET) == load_library_parquet.version
+def test_get_ingest_config(load_ingest: library.Config):
```

Comment:
does this test use the pytest fixture above, `load_ingest`? If so, I'm a bit confused what exactly we are comparing here

Reply:
It does.

Yeah, this is maybe a bit sloppy by me - I figured easiest way to mock having a config in s3 was to just run the ingest process, but this is not great practice. recipe tests really shouldn't depend on `ingest` code having run successfully, since that same ingest code is relying on other recipe functions. 

There's some argument that 
- "write" recipe code is tested on its own
- "write" ingest code tested on already tested recipe code
- "read" recipe code is tested using tested ingest code

Isn't horribly improper, since it's not circular... But probably best to keep them separate entirely. I'll rework this to not use ingest code

Reply:
So the idea with this function was 
- use ingest to dump a config in s3
- test that this config is read successfully